pennylane/devices/tests/test_gates.py

```diff
@@ -64,7 +64,7 @@
     "CPhaseShift00": qml.CPhaseShift00(0, wires=[0, 1]),
     "CPhaseShift01": qml.CPhaseShift01(0, wires=[0, 1]),
     "CPhaseShift10": qml.CPhaseShift10(0, wires=[0, 1]),
-    "QubitStateVector": qml.QubitStateVector(np.array([1.0, 0.0]), wires=[0]),
+    "StatePrep": qml.StatePrep(np.array([1.0, 0.0]), wires=[0]),
```

Comment:
```suggestion
    "QubitStateVector": qml.QubitStateVector(np.array([1.0, 0.0]), wires=[0]),
    "StatePrep": qml.StatePrep(np.array([1.0, 0.0]), wires=[0]),
```

Worth continuing to test until we actually remove `QubitStateVector`.

Reply:
👍🏼